lib/cmd/fh3/appdata/export/list.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+var fs = require('fs')
+  , fhreq = require('../../../../utils/request.js')
+  , common = require('../../../../common');
+
+module.exports = {
+  'desc': 'List all current and historic export jobs given an application and environment id',
+  'examples': [{
+    cmd: 'fhc addpata export list --appId=5ujx2eifvzaudq43nw4nmvcu --envId=live',
+    desc: 'List all current and historic export jobs given an application and environment id'
+  }],
+  'demand': ['envId', 'appId'],
+  'alias': {},
+  'describe': {
+    'envId': 'Environment id',
+    'appId': 'Application id'
+  },
+  'method': 'get',
+  'customCmd': function (params, cb) {
+    var url = 'api/v2/appdata/' + params.envId + '/' + params.appId + '/data/export';
+
+    common.doGetApiCall(fhreq.getFeedHenryUrl(), url, 'Error listing exports', function (err, data) {
+      if (err) {
+        return cb(err);
+      }
+
+      params._table = common.createTableForAppData(data);
+      return cb(null, params);
+    });
+  }
```

Comment:
In the new command structure you can override the url with `url = function(params){}` and return a string, no need for `customCmd`.

It might not support `--json` if it's done like this.